Cooper/DAO/ChatDAO.cs

```diff
@@ -39,10 +39,10 @@ public ChatDb Get(long id)
         {
             ChatDb chat = null;
 
-            EntityORM entity = crud.Read(id, idColumn, attributes, table);
+            var entity = (List<EntityORM>)(crud.Read(table, attributes, new DbTools.WhereRequest[] { new DbTools.WhereRequest(idColumn, DbTools.RequestOperator.Equal, id) }));
```

Comment:
don't use var. it is not obvious what type will come
and here is a list, but "entity" is for single entity, use "entities"